executor/src/witgen/global_constraints.rs

```diff
@@ -264,6 +264,9 @@ fn propagate_constraints<T: FieldElement>(
                 }
             }
         }
+        Identity::PhantomPermutation(..) | Identity::PhantomLookup(..) => {
+            // phantom identities do not provide any constraints
```

Comment:
They actually do, don't they? I mean we assume they are mapped properly, but if they are, this is where we get the range constraints from.

Reply:
Oh, right :)